_transactions.md

```diff
@@ -267,6 +267,140 @@ url        | The URL of the 3DSecure confirmation request.
 
 A webpage for 3DSecure confirmation for the user to interact with.
 
+## Beneficiary Information
+
+> Example of a transaction creation payload including `beneficiary` and `purpose` fields:
+
+```json
+{
+  "beneficiary": {
+    "relationship": "child",
+    "name": "Harper Wilson",
+    "address": {
+      "line1": "Bar",
+      "zipCode": "12345",
+      "city": "Braga",
+      "country": "PT",
+      "state": "PT-03"
+    }
+  },
+  "purpose": "donations",
+  "denomination": {
+    "amount": "3000",
+    "currency": "USD"
+  },
+  "destination": "invite-user@gmail.com"
+}
+```
+
+When creating a transaction, it may be required to send additional information
+regarding the beneficiary and purpose of the transaction.
+This is required if **both** the following conditions are met:
+
+- The transaction is either a withdrawal to an external account (crypto networks, SEPA bank accounts, etc.) or a transfer to another user; and
+- The transaction amount is over _$3000 USD_ (or over $1000 USD, if the origin user is from Arizona).
+
+<aside class="notice">
+  The threshold value is calculated from the <a href="#normalized">normalized</a> amount of the transaction,
+  which can differ from the amount specified in the <code>denomination.amount</code> field
+  if the transaction implies a currency conversion.
+</aside>
+
+The beneficiary information should be provided in the `beneficiary` object of the request payload.
+The `beneficiary.relationship` field must always be filled whenever beneficiary information is required.
+The other fields can be omitted if the data is already present in the Uphold platform
+(e.g. if the transaction is to an existing Uphold user, or the `beneficiary.relationship` is set to "myself").
+
+The `relationship` field is a string reflecting the beneficiary's relationship to the transaction originator.
+Two particular types of transactions are recognized by the platform:
+business transactions (indicated by a `relationship` field with the value `business`)
+and personal transactions (indicated by the values `child`, `co_worker`, `friend`, `parent`, or `sibling`).
+The special value `myself` can also be used when applicable.
+
+Depending on the conditions of the transaction, additional fields may be required
+— namely, the beneficiary `name` and `address` (with subfields `city`, `country`, `line1`, `line2`, `state`, `zipCode`),
```

Comment:
Should we state which fields are mandatory here?

Reply:
We don't want to duplicate the logic of the code into the docs, both to minimize the possibility of them becoming out of sync, and to avoid making the docs too complex (since the actual requirements depend on the interaction of various conditions). The goal here was to explain the behavior of the API, and let the API responses provide the specific conditions.